lib/bounding-boxes.ts

```diff
@@ -42,6 +42,73 @@ const alphaTwoCountryCodeBoundingBoxData: Record<
   string,
   BoundingBox | undefined
 > = {
+  AD: [1.41, 42.42, 1.79, 42.66],
```

Comment:
I'll sort the list in another PR later. I just want to include these at the top so that if you're curious which countries were included that's clear. We'll make the order pleasant later.

For chains of islands, the bounding box includes all of the islands so that's why some of these are really big. There's a couple SC2Tracker countries that weren't zoomed in on properly because they were missing from this list (Andorra, Faroe Islands, the Holy See (although that bonding box is hard to get right for some reason), Cabo Verde, etc.)